include/library/spdm_common_lib.h

```diff
@@ -898,6 +896,7 @@ bool libspdm_get_local_cert_chain_buffer(void *spdm_context,
  * @retval false Local used certificate chain data without spdm_cert_chain_t header is not found.
  **/
 bool libspdm_get_local_cert_chain_data(void *spdm_context,
+                                       uint8_t slot_id,
```

Comment:
`libspdm_get_local_cert_chain_data` is an unused function, so I am also fine if the entire function is removed.